### PR TITLE
Fix interface sliding on mobile

### PR DESCRIPTION
**Pull request by Abyss777**
The best way is `display:none` but in most browsers images are not loaded.

Just moved outside visible area.

### CODE REVIEW

File: web/load.js [hunk 1]
```diff
@@ -21,7 +21,7 @@
         svg.setAttribute('id', id);
         svg.setAttribute('data', file);
         svg.setAttribute('type', 'image/svg+xml');
-        svg.setAttribute('style', 'visibility:hidden');
+        svg.setAttribute('style', 'visibility:hidden;position:absolute;left:-40px;');
```
Comment (tananaev) [inline web/load.js]: Is the `left:-40px` required, or `position:absolute` would be enough? I just don't like that we have some hardcoded number there. If it's required, what if we put `0px`?

Comment (tananaev) [inline web/load.js]: OK, let's use `-100px` for now.

**Comment by Abyss777** [comment]
Replaced commit with `-100px`

**Comment by tananaev** [comment]
Merged. Thanks for pull request.

**Comment by Abyss777** [comment]
I guess "Merge" button just was not pressed :-)

**Comment by tananaev** [comment]
Yes, looks like I forgot to click the button. Too many pull requests :)

**Pull request merged**